src/containers/control/add/index.js

```diff
@@ -131,10 +131,10 @@ export default class ControlView extends React.Component {
         state: controlState
       })
     } else {
-      this.props.saveControl(this.props.params.id, controlState, {
-        onSuccess: () => hashHistory.goBack(),
-        onFailure: () => { /* alert('Kunne ikke lagre kontroll') */ }
-      })
+      console.log(this.props.params)
+      this.props.saveControl(controlState, { onSuccess: () => hashHistory.goBack(),
+                                             onFailure: () => window.alert('Kunne ikke lagre kontroll') },
```

Comment:
this needs to be cleaned up, format like this:
this.whatever(1. state, {
  onSuccess: ???,
  onFailure: ????
})